pylint_odoo/checkers/modules_odoo.py

```diff
@@ -351,6 +357,14 @@ def visit_import(self, node):
             if isinstance(node.scope(), astroid.Module):
                 self._check_imported_packages(node, name)
 
+    @utils.check_messages('except-pass')
+    def visit_tryexcept(self, node):
+        """Visit block try except"""
+        for handler in node.handlers:
```

Comment:
Here you could validate if the except has a ` as ` in order to skip the check because the new variable could be used.
It this variable is not used then we will have a `variable-unused`

This is the easiest way to skip the check using the @lasley recommendation